### ISSUE TITLE
can't acces the website 

### ISSUE THREAD

**Issue by ninsat** [emmanuel-florent/bitcoinpages-home#1]
can't acces the website

**Comment by eflorent2020** [comment]
yes it's a work in progress, basic part of my secret plan: an ads search engine that leverage a Bitcoin wallet I'm writting. This project leverage the Ghost blog engine and thus do much more that the code it contains.
